esp/platform/espcontext.cpp

```diff
@@ -586,12 +586,28 @@ void CEspContext::updateTraceSummaryHeader()
     if (m_txSummary)
     {
         m_txSummary->set("activeReqs", m_active);
-        m_txSummary->set("user", VStringBuffer("%s%s%s", (queryUserId() ? queryUserId() : ""), (m_peer.length() ? "@" : ""), m_peer.str()).str());
+        VStringBuffer user("%s%s%s", (queryUserId() ? queryUserId() : ""), (m_peer.length() ? "@" : ""), m_peer.str());
+        if (!user.isEmpty())
+            m_txSummary->set("user", user.str());
 
-        VStringBuffer reqSummury("%s %s.%s", httpMethod.get(), m_servName.str(), servMethod.get());
+        VStringBuffer reqSummary("%s", httpMethod.isEmpty() ? "" : httpMethod.get());
+        if (!m_servName.isEmpty() || !servMethod.isEmpty())
+        {
+            if (!reqSummary.isEmpty())
+                reqSummary.append(" ");
+            if (!m_servName.isEmpty())
+                reqSummary.append(m_servName.str());
+            if (!servMethod.isEmpty())
```

Comment:
You must ask: "is it possible servName.isEmpty and !servMethod.isEmpty?" In that case, the reqsummary contains ".servMethod" which indicates a method name (not a service name): servMethod.